h2o-core/src/test/java/water/rapids/RapidsTest.java

```diff
@@ -629,6 +631,53 @@ static boolean checkSaneFrame_impl() {
     }
   }
 
+  @Test public void testNaOmit() {  // code from Nidhi
```

Comment:
If this test is for the `AstNaOmit` class, then it should go into the `AstNaOmitTest` class. This serves several purposes: allows us to quickly find which classes has or don't have tests; prevent the `RapidsTest` to grow beyond measure; better reporting in case the test fails.